tests/common/utils.rs

```diff
@@ -40,6 +40,14 @@ pub fn event_disputed(channel_id: ChannelIdOf<Test>, state: StateOf<Test>) {
 	);
 }
 
+/// Checks that the last event was a `Progressed` event with the given args.
+pub fn event_progressed(channel_id: ChannelIdOf<Test>) {
```

Comment:
Nitpick:
```suggestion
pub fn assert_event_progressed(channel_id: ChannelIdOf<Test>) {
```

Reply:
Applied, also for the other event types.